src/main/java/frc/robot/constants/ArmConstants.java

```diff
@@ -1,21 +1,21 @@
 package frc.robot.constants;
 
 public class ArmConstants {
-  // arm ids - TBD
-  public static final int motorID = 1;
+  // TODO: arm ids - TBD (fake using 5 -- 1-4 have been taken)
+  public static final int motorID = 5;
 
-  // pid values - TBD
-  public static final double kP = -1;
-  public static final double kI = -1;
-  public static final double kD = -1;
-  public static final double kTolerance = -1;
-  public static final double minMotorPower = -1;
-  public static final double maxMotorPower = -1;
+  // TODO: PID values - TBD (fake values for now)
+  public static final double kP = 1.0;
+  public static final double kI = 0.0;
+  public static final double kD = 0.0;
+  public static final double kTolerance = 0.02;
+  public static final double minMotorPower = -1.0;
+  public static final double maxMotorPower = +1.0;
```

Comment:
for safety I would use lower values

Reply:
min/max power reduced to 0.05.

Reply:
kTolerance of 0.02 radian is about 1 degree.

Reply:
kTolerance is fine, mainly meant min/max reduced. Though that being said, to be more clear, you can do Units.degreesToRadians(1) which is easier to understand and emphasizes the units used.